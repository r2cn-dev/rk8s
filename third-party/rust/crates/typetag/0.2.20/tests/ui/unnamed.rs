#[typetag::serde]
pub trait Trait {}

#[typetag::serde]
impl Trait for (u8, u8) {}

fn main() {}
