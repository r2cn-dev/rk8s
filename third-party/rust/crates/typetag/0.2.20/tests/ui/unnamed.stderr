error: use #[typetag::serde(name = "...")] to specify a unique name
 --> tests/ui/unnamed.rs:5:16
  |
5 | impl Trait for (u8, u8) {}
  |                ^^^^^^^^
